p5/visualTests/sanity_testing.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import os
+from subprocess import run, PIPE
+
+e = SystemExit()
+
+dirname = os.path.dirname(__file__)
```

Comment:
I think `os.path.dirname(os.path.realpath(__file__))` is more robust, since it handles relative directories as well.
Source: https://stackoverflow.com/questions/5137497/find-current-directory-and-files-directory

Reply:
Great, didn't know that and it was breaking when I ran sanity_testing from its directory. Thanks